azure-pipelines-release.yml

```diff
@@ -0,0 +1,730 @@
+parameters:
```

Comment:
if this file replaces azure-pipelines.yml then azure-pipelines.yml should be deleted as part of this PR